app/jobs/voice_otp_sender_job.rb

```diff
@@ -20,15 +20,37 @@ def send_otp(twilio_service, code, phone)
     )
   end
 
-  def twimlet_url(code)
-    "https://twimlets.com/message?#{twimlet_query_string(code)}"
+  def twimlet_url(code) # rubocop:disable Metrics/MethodLength
```

Comment:
if I inline the `repeat = message_repeat(code)` part to shorten and get under the line length limit, reek gets mad about repeat statements, this seemed like a decent compromise to me